spikeextractors/extractors/neoextractors/blackrockextractor.py

```diff
@@ -53,3 +55,8 @@ class BlackrockSortingExtractor(NeoBaseSortingExtractor):
     mode = 'file'
     installed = HAVE_NEO
     NeoRawIOClass = 'BlackrockRawIO'
+
+    def __init__(self, filename: PathType, nsx_to_load: Optional[int] = None,
```

Comment:
@luiztauffer isn't the sorted daata in the `nev` file? Why the `nsx_to_load` arg then?

Reply:
its needed for extracting the sampling rate it seems, I think this is happening in NEO

Reply:
To be more specific: If have multiple blackrock files (e.g. nev, ns3, ns6) when reading from nev, and nsx_to_load is not specified, I think NEO will search for the first stream it can use to get the sampling rate. At least that’s the behavior I observed here. 
Also, see: https://github.com/SpikeInterface/spikeextractors/issues/543